operations/credhub-development.yml

```diff
@@ -0,0 +1,16 @@
+- type: replace
+  path: /instance_groups/name=bosh/jobs/name=atc/properties/credhub?
+  value:
+    url: "https://((developmentbosh_target)):8844/api/"
+    tls:
+      insecure_skip_verify: true
```

Comment:
If we're setting a ca cert, why do we also need to skip verification?

Reply:
yeah @jontours we can leave this one in. We had to disable TLS previously while we were testing because it was for the RDS instances. But eventually we should add those certs in there too.